subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/internal/DefaultJvmEcosystemAttributesDetails.java

```diff
@@ -51,6 +52,18 @@ public JvmEcosystemAttributesDetails runtimeUsage() {
         return this;
     }
 
+    @Override
+    public JvmEcosystemAttributesDetails apiCompileView() {
+        attributes.attribute(CompileView.VIEW_ATTRIBUTE, objectFactory.named(CompileView.class, CompileView.JAVA_API));
+        return this;
+    }
+
+    @Override
+    public JvmEcosystemAttributesDetails completeInternalView() {
```

Comment:
Maybe just stick with `internalCompileView` here and in `JvmEcosystemAttributeDetails`.  Why create a new term?

Reply:
Good catch. This was a remnant of the old naming scheme.